Mathlib/RingTheory/TensorProduct/MonoidAlgebra.lean

```diff
@@ -0,0 +1,82 @@
+/-
+Copyright (c) 2025 Michał Mrugała. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michał Mrugała
+-/
+import Mathlib.Algebra.MonoidAlgebra.Basic
+import Mathlib.RingTheory.IsTensorProduct
+
+/-!
+# Monoid algebras are invariant under base change
+
+In this file we show that monoid algebras are stable under pushout.
+
+## TODO
+
+Additivise
+-/
+
+noncomputable section
+
+open TensorProduct
+
+namespace MonoidAlgebra
+variable {R M S A B : Type*} [CommSemiring R] [CommSemiring S] [Algebra R S] [CommMonoid M]
+variable [CommSemiring A] [Algebra R A] [CommSemiring B] [Algebra R B]
+
+variable (R A B) in
+/-- Tensoring `MonoidAlgebra R M` on the left by an `R`-algebra `A` is algebraically
+equivalent to `MonoidAlgebra A M`. -/
+noncomputable def tensorEquiv : A ⊗[R] MonoidAlgebra B M ≃ₐ[A] MonoidAlgebra (A ⊗[R] B) M := by
+  refine .ofAlgHom
+    (Algebra.TensorProduct.lift
+      ((IsScalarTower.toAlgHom A (A ⊗[R] B) _).comp Algebra.TensorProduct.includeLeft)
+      (mapRangeAlgHom _ Algebra.TensorProduct.includeRight)
+      (fun p n => .all _ _))
```

Comment:
```suggestion
      (fun p n ↦ .all _ _))
```
There is no preference between the two in mathlib but I would prefer there is at least consistency in the same definition. If you want you can also change the other `↦` to `=>` instead.